kbs/config/kubernetes/README.md

```diff
@@ -44,6 +44,24 @@ kustomize edit add resource ingress.yaml
 popd
 ```
 
+## Optional: Expose KBS using Nodeport
+
+If you would like to expose KBS service using Nodeport then add the following snippet to the `overlays/kustomization.yaml` file in the `patches:` section:
```

Comment:
My preference would be to have a dedicated kustomization for this, e.g, `overlays/nodeport` so that manual editing can be avoided.

Reply:
Done, PTAL.